timeseries/src/autogluon/timeseries/models/presets.py

```diff
@@ -244,7 +192,7 @@ def get_preset_models(
                 f"Keys of the `hyperparameters` dictionary must be strings or types, received {type(model)}."
             )
 
-        for model_hps in hyperparameters[model]:
+        for model_hps in hyperparameter_dict[model]:
             ag_args = model_hps.pop(constants.AG_ARGS, {})
```

Comment:
Thanks, the PR looks great! I guess we can also move this logic closer to the `TimeSeriesTrainer` at some point. IMO, there should be just one "preset_configs" file that contains 
- equivalent of the `get_default_hps` method
- preset configurations

Everything else feels like it should be a part of the trainer.